modules/oadp-self-signed-certificate.adoc

```diff
@@ -38,4 +38,4 @@ spec:
 ...
 ----
 <1> Specify the Base46-encoded CA certificate string.
-<2> Must be `false` to disable SSL/TLS security.
+<2> The `insecureSkipTLSVerify` config can be set to either `"true"` or `"false"`. If set to `"true"`, TLS/SSL security is disabled. If set to `"false"`, TLS/SSL security is enabled.
```

Comment:
* Per the SSG, it should be "SSL/TLS", not "TLS/SSL": https://redhat-documentation.github.io/supplementary-style-guide/#ssl-tls
* "config" should likely be spelled out as "configuration", or changed to "field" or something